src/app/components/widget/widget.factory.js

```diff
@@ -47,13 +47,33 @@
         CgroupMemoryUtilizationMetricDataModel,
         config) {
 
-        var onSettingsClose = function(resultFromModal, widgetModel) {
-            if (typeof resultFromModal !== 'undefined'){
+        var onSettingsClose = function (resultFromModal, widgetModel) {
+            if (typeof resultFromModal !== 'undefined') {
                 widgetModel.filter = resultFromModal.filter;
             }
         };
         var definitions = [
             {
+                name: 'nfs4.client',
```

Comment:
New widget definition is probably not part of the feature. Also, nfs4.* would not work well with this data model.